### PR TITLE
chore(kuma-cp) rename 'kumactl generate dataplane-token' CLI arg

### PR DESCRIPTION
**Pull request by lobkovilya**
Signed-off-by: Ilya Lobkov <ilya.lobkov@konghq.com>

### Summary

rename `--dataplane` to `--name`

### CODE REVIEW

File: app/kumactl/cmd/generate/generate_dataplane_token.go [hunk 1]
```diff
@@ -13,7 +13,8 @@ type generateDataplaneTokenContext struct {
 	*kumactl_cmd.RootContext
 
 	args struct {
-		dataplane string
+		dataplane string // Deprecated: remove in next major version of Kuma 1.1
```
Comment (nickolaev) [inline app/kumactl/cmd/generate/generate_dataplane_token.go:16]: We have to find a way to track these deprecations. Either we have a fixed tag, like `Deprecated:` or we track in a dedicated doc.

**Review by nickolaev — APPROVED** [review]

**Review by jakubdyszkiewicz — APPROVED** [review]

**Comment by jakubdyszkiewicz** [comment]
Make sure to update docs

**Review by nickolaev — CHANGES_REQUESTED** [review]
Please provide a link to the updated docs in the PR descripton.

**Comment by lobkovilya** [comment]
That's weird, I looked through all documentation, but it's already correct:
```
kumactl generate dataplane-token \
  --name dp-echo-1 \
  --mesh default \
  --tag kuma.io/service=backend,backend-admin > /tmp/kuma-dp-echo1-token
```

**Review by nickolaev — APPROVED** [review]

**Pull request merged**